### DISCUSSION TITLE
Can't connect to SSH

### DISCUSSION THREAD

**Discussion by JelgerDijkgraaf** [mcguirepr89/BirdNET-Pi#1267] [Q&A] [answered]
I have followed the guide exactly, even with the names. But I either get this: `PS C:\> ssh birder@birdnetpi1.local
ssh: Could not resolve hostname birdnetpi1.local: No such host is known.` or `PS C:\> ssh pi@<rpi ip gotten via advanced ip scanner>
ssh: connect to host <ip> port 22: Connection timed out`.

I'm working on windows with a raspberry 4 model B.
When I boot up the rpi the green led blinks for a while, but after like 8 min it stops. Could this be because of a boot up failure? If so, what could it be? Files like config and cmdline are present. 

This is my first time using a raspberry pi, hoping I gave enough information and someone can help:)

update: I don't know exactly why it went better now but I used Putty and an ethernet cable between my laptop and the rpi. Now I can login to the rpi but when installing it it ends up with `Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: tflite_runtime-2.6.0-cp39-none-linux_aarch64.whl is not a supported wheel on this platform.
The installation exited unsuccessfully.`
The birdnet1.local site shows the Apache2 Debian Default Page.

I used the 64 lite bookworm version. Also tried other ones like the bullseye but with that one I couldn't even connect to it.

**Comment by mattgolde** [comment, marked as answer]
Looks like an installation problem, try writing the OS to the card again, using the raspberry pi installer - and check you have the 64 bit lite for bullseye for rpi4B if using this (mcguirepr89) distribution.
Bookworm will likely not work.
There is another (newer) birdnet distribution (nachtzuster) that uses bookworm if you prefer.

**Reply by JelgerDijkgraaf** [reply]
Thanks!! I tried the nachtzuster version and it worked first try.
